.devfile

```diff
@@ -0,0 +1,21 @@
+specVersion: 0.0.1
+name: che-in-che
+projects:
+  - name: che
+    source:
+      type: git
+      location: 'https://github.com/eclipse/che.git'
+tools:
+  - name: theia-editor
+    type: cheEditor
+    id: org.eclipse.che.editor.theia:1.0.0
+  - name: exec-plugin
+    type: chePlugin
+    id: che-machine-exec-plugin:0.0.1
+commands:
+  - name: dummy-stub-command
```

Comment:
Why do we need `dummy-stub-command` for work Che in Che?

Reply:
Because commands is a mandatory section. I'll create an issue to make them optional. But in general, we need to populate this list with a lot of useful commands. Here we can elaborate on which one we will be able to introduce with the current level of devfile support.

Reply:
We definitely need to create an issue because commands provided by plugins may be enough for development. If we all agree on that - maybe you can do it in this PR? I guess there is one line to change. @mshaposhnik Can you point what should be changed to make commands optional?

Reply:
yes. remove the https://github.com/eclipse/che/blob/master/wsmaster/che-core-api-devfile/src/main/resources/schema/devfile.json#L16 line.

Reply:
https://github.com/eclipse/che/pull/12308